README.md

```diff
@@ -17,7 +17,7 @@ Welcome to the Yoti Ruby SDK. This repository contains the tools you need to qui
 ## An Architectural view
 
 To integrate your application with Yoti, your back-end must expose a GET endpoint that Yoti will use to forward tokens.
-The endpoint can be configured in your Yoti Dashboard when you create/update your application. It can be found in the Integration section under the Callback URL name.
+The endpoint can be configured in your [Yoti Dashboard](https://www.yoti.com/dashboard) when you create/update your application. To see an example of how this is configured, see the [Running the Examples](#running-the-examples) section.
```

Comment:
My pickiness goes further...
The endpoint *is* configured in *the* [Yoti Dashboard](https://www.yoti.com/dashboard), *where* you create/update your application. To see an example --of how this is configured--, see --the-- [Running the Examples](#running-the-examples) --section--.